.gitmodules

```diff
@@ -77,3 +77,6 @@
 [submodule "platform/broadcom/sonic-platform-modules-mitac"]
 	path = platform/broadcom/sonic-platform-modules-mitac
 	url = https://github.com/MiTAC-EBU/sonic-platform-modules-mitac.git
+[submodule "platform/centec/sonic-platform-modules-e582"]
+	path = platform/centec/sonic-platform-modules-e582
+	url = https://github.com/CentecNetworks/sonic-platform-modules-e582
```

Comment:
hi as we have discussed in community meeting, new platform module should be in tree. can you pull the source code into sonic-buildimage. Thanks.

Reply:
thanks, i will close this PR and create a new PR after these changes are done.